### ISSUE TITLE
problem with CLI-mode with latest xaml-fix 

### ISSUE THREAD

**Issue by xmatthias** [yck1509/ConfuserEx#201]
after my previous issue #195 was fixed (c04d4f6b7ca6c5a2509613146b43f7158759df61) i thought everything works fine.
It works fine when run from GUI - however, running it from the comandline supplying the same crproj (works when run from GUI) results in the following error:
```
 [INFO] ConfuserEx v0.4.0-74-gc04d4f6 Copyright (C) Ki 2014
 [INFO] Running on Microsoft Windows NT 6.1.7601 Service Pack 1, .NET Framework v4.0.30319.34209, 64 bits
[DEBUG] Discovering plugins...
 [INFO] Discovered 10 protections, 1 packers.
[DEBUG] Resolving component dependency...
 [INFO] Loading input modules...
 [INFO] Loading 'Confuser_Demo.exe'...
 [INFO] Initializing...
[DEBUG] Building pipeline...
 [INFO] Resolving dependencies...
[DEBUG] Checking Strong Name...
[DEBUG] Creating global .cctors...
[DEBUG] Watermarking...
[DEBUG] Executing 'Name analysis' phase...
[DEBUG] Building VTables & identifier list...
[DEBUG] Analyzing...
[DEBUG] WPF found, enabling compatibility.
[ERROR] Unknown error occurred.
Exception: System.UriFormatException: Ungltiger URI: Es wurde ein ungltiger Anschluss angegeben.
   bei System.Uri.CreateThis(String uri, Boolean dontEscape, UriKind uriKind)
   bei Confuser.Renamer.BAML.BAMLAnalyzer.ProcessConverter(PropertyWithConverterRecord rec, TypeDef type) in c:\projects\confuserex\Confuser.Renamer\BAML\BAMLAnalyzer.cs:Zeile 506.
   bei Confuser.Renamer.BAML.BAMLAnalyzer.ProcessElementBody(BamlElement root, BamlElement elem) in c:\projects\confuserex\Confuser.Renamer\BAML\BAMLAnalyzer.cs:Zeile 337.
   bei Confuser.Renamer.BAML.BAMLAnalyzer.ProcessBAMLElement(BamlElement root, BamlElement elem) in c:\projects\confuserex\Confuser.Renamer\BAML\BAMLAnalyzer.cs:Zeile 260.
   bei Confuser.Renamer.BAML.BAMLAnalyzer.Analyze(ModuleDefMD module, String bamlName, Byte[] data) in c:\projects\confuserex\Confuser.Renamer\BAML\BAMLAnalyzer.cs:Zeile 120.
   bei Confuser.Renamer.Analyzers.WPFAnalyzer.AnalyzeResources(ConfuserContext context, INameService service, ModuleDefMD module) in c:\projects\confuserex\Confuser.Renamer\Analyzers\WPFAnalyzer.cs:Zeile 277.
   bei Confuser.Renamer.AnalyzePhase.Analyze(NameService service, ConfuserContext context, ProtectionParameters parameters, IDnlibDef def, Boolean runAnalyzer) in c:\projects\confuserex\Confuser.Renamer\AnalyzePhase.cs:Zeile 104.
   bei Confuser.Renamer.AnalyzePhase.Execute(ConfuserContext context, ProtectionParameters parameters) in c:\projects\confuserex\Confuser.Renamer\AnalyzePhase.cs:Zeile 55.
   bei Confuser.Core.ProtectionPipeline.ExecuteStage(PipelineStage stage, Action`1 func, Func`1 targets, ConfuserContext context) in c:\projects\confuserex\Confuser.Core\ProtectionPipeline.cs:Zeile 138.
   bei Confuser.Core.ConfuserEngine.RunPipeline(ProtectionPipeline pipeline, ConfuserContext context) in c:\projects\confuserex\Confuser.Core\ConfuserEngine.cs:Zeile 208.
   bei Confuser.Core.ConfuserEngine.RunInternal(ConfuserParameters parameters, CancellationToken token) in c:\projects\confuserex\Confuser.Core\ConfuserEngine.cs:Zeile 162.
Failed at 09:15, 0:01 elapsed.
```

unfortunately my system is in german - however Ungültiger URI should be translated as "invalid URI" ...

if you need more details about it letme know.

sample project (same as for issue #195)
https://mega.co.nz/#!yV50gYIL!p6qLGXejdXlAhRDS6tBw35qt6dXw8uOyBS9Z3LMClJw

edits: formatting

**Comment by xmatthias** [comment]
fixed myself with pull-request #202

**Comment by yck1509** [comment]
Fixed with #210.
